mods/cnc/maps/gdi03/rules.yaml

```diff
@@ -4,7 +4,7 @@ World:
 	MusicPlaylist:
 		StartingMusic: crep226m
 	MissionData:
-		Briefing: Build up forces to destroy Nod base.\n\nOnce all Nod SAM sites are neutralized then air support will be provided to combat obstacles such as turrets.\n\nDestroy all units and structures to complete the mission objective.
+		Briefing: Build up your forces to destroy the Nod base.\n\nOnce all Nod SAM sites are neutralized, air support will deal with ground defenses.\n\nDestroy all units and structures to complete the mission.
```

Comment:
This change suggests that airstrikes will automatically destroy the defenses, which isn't the case.